packages/discord.js/src/structures/StageInstance.js

```diff
@@ -83,6 +93,15 @@ class StageInstance extends Base {
     return this.client.guilds.resolve(this.guildId);
   }
 
+  /**
+   * The associated scheduled event of this stage instance
+   * @type {?GuildScheduledEvent}
+   * @readonly
+   */
+  get scheduledEvent() {
+    return this.guild?.scheduledEvents.resolve(this.guildScheduledEventId);
+  }
```

Comment:
So doesn't this possibly return `undefined`? Might want to chuck a `?? null` after it maybe?

Reply:
`this.guild` is `null` if not found so it should never be `undefined`.

Reply:
if `this.guild` is null then `this.guild?.scheduledEvents.resolve(this.guildScheduledEventId)` will return `undefined`. no?

Reply:
Yeah it's been fixed.